src/providers/tpm_provider/utils.rs

```diff
@@ -233,13 +234,13 @@ pub fn signature_data_to_bytes(data: SignatureData, key_attributes: Attributes)
 }
 
 pub fn parsec_to_tpm_signature(
-    data: Vec<u8>,
+    data: &Zeroizing<Vec<u8>>,
```

Comment:
Does it work if you remove the reference? Since `Zeroizing` implements `Deref`, I think that `to_vec` takes the reference automatically.

Reply:
I think it's actually ok to pass the `Zeroizing` ownership in here, we don't need the signature data in the buffer after this conversion. Ideally the whole `Zeroizing` will be passed into the TSS signature crate.